applications/GeoMechanicsApplication/tests/cpp_tests/test_axisymmetric_stress_state.cpp

```diff
@@ -17,34 +17,21 @@
 #include "geometries/geometry.h"
 #include "includes/checks.h"
 #include "testing/testing.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
 #include <boost/numeric/ublas/assignment.hpp>
 
 using namespace Kratos;
 
 namespace Kratos::Testing
 {
 
-ModelPart& CreateModelPart(Model& rModel)
-{
-    ModelPart& result = rModel.CreateModelPart("Main");
-
-    result.CreateNewNode(1, 0.0, 0.0, 0.0);
-    result.CreateNewNode(2, 1.0, 0.0, 0.0);
-    result.CreateNewNode(3, 1.0, 1.0, 0.0);
-
-    std::vector<ModelPart::IndexType> node_ids{1, 2, 3};
-    result.CreateNewElement("UPwSmallStrainElement2D3N", 1, node_ids, result.CreateNewProperties(0));
-
-    return result;
-}
-
 KRATOS_TEST_CASE_IN_SUITE(CalculateBMatrixWithValidGeometryReturnsCorrectResults, KratosGeoMechanicsFastSuite)
 {
     const std::unique_ptr<StressStatePolicy> p_stress_state_policy =
         std::make_unique<AxisymmetricStressState>();
 
     Model model;
-    auto& model_part = CreateModelPart(model);
+    auto& model_part = ModelSetupUtilities::CreateModelPartWithASingle2D3NElement(model);
```

Comment:
Should the variable name start with r?

Reply:
On behalf of @indigocoral it has been done.